apps/judicial-system/web/src/routes/Court/InvestigationCase/HearingArrangements/HearingArrangements.strings.ts

```diff
@@ -136,7 +136,8 @@ export const icHearingArrangements = {
     secondaryButtonText: {
       id:
         'judicial.system.core:investigation_cases_hearing_arrangements.modal.secondary_button_text',
-      defaultMessage: 'Nei',
+      defaultMessage:
```

Comment:
Adding a param to a contentful string requires the id to be updated.

Reply:
Even if the contentful string hadn't been added yet?

Reply:
Discussed offline. The string is not yet contentful so it is fine to not update the id